aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,15 +1,16 @@
 ---
 remote_endpoint: ~
-name: "v1.14"
+name: "v1.16"
 proposals:
   - name: step_1_upgrade_framework
     metadata:
-      title: "Multi-step proposal to upgrade mainnet framework to v1.14"
-      description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.13"
+      title: "Multi-step proposal to upgrade mainnet framework to v1.16"
+      description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.16"
     execution_mode: MultiStep
     update_sequence:
       - Framework:
```

Comment:
Actually, I think the correct order is to do the gas schedule upgrade first and then the framework. This is because the framework could contain new features that depend on the new version of the gas schedule, but not vice versa.